bin/update/deploy_base.py

```diff
@@ -133,7 +141,10 @@ def update_info(ctx):
 def peep_install(ctx):
     """Install things using peep."""
     with ctx.lcd(settings.SRC_DIR):
-        ctx.local('../venv/bin/peep install -r requirements/compiled.txt')
+        ctx.local(peep_install_cmd('prod'))
+        # update any newly installed libs to be relocatable
+        # this call is idempotent
+        ctx.local('virtualenv-2.7 --relocatable ../venv')
```

Comment:
Yeah. We unfortunately don't have a choice at the moment unless we rsync the virutalenv separately to a path on the webheads that exactly matches the one on the admin node.